packages/polyfills/types/dom.d.ts

```diff
@@ -13,10 +12,54 @@ interface ImportMeta {
     url: string
 }
 
+// #region Web Components
 interface ShadowRoot {
     adoptedStyleSheets?: readonly CSSStyleSheet[]
 }
 interface CSSStyleSheet {
     replace(text: string): Promise<void>
     replaceSync(text: string): void
 }
+// #endregion
+
+// #region Trusted Types
+interface TrustedHTML {
```

Comment:
need eslint rule? prefer-trusted-html

Reply:
No need, and you also cannot make a linter rule for it.